forge-gui/res/cardsfolder/r/rite_of_ruin.txt

```diff
@@ -2,12 +2,12 @@ Name:Rite of Ruin
 ManaCost:5 R R
 Types:Sorcery
 A:SP$ GenericChoice | Cost$ 5 R R | Choices$ ChooseC1L2A3,ChooseL1C2A3,ChooseA1L2C3,ChooseA1C2L3,ChooseC1A2L3,ChooseL1A2C3 | Defined$ You | StackDescription$ SpellDescription | SpellDescription$ Choose an order for artifacts, creatures, and lands. Each player sacrifices one permanent of the first type, sacrifices two of the second type, then sacrifices three of the third type.
-SVar:ChooseC1L2A3:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ ChooseC1 | SubAbility$ SacC1L2A3 | SpellDescription$ Creature,Land,Artifact
-SVar:ChooseL1C2A3:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ ChooseL1 | SubAbility$ SacL1C2A3 | SpellDescription$ Land,Creature,Artifact
-SVar:ChooseA1L2C3:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ ChooseA1 | SubAbility$ SacA1L2C3 | SpellDescription$ Artifact,Land,Creature
-SVar:ChooseA1C2L3:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ ChooseA1 | SubAbility$ SacA1C2L3 | SpellDescription$ Artifact,Creature,Land
-SVar:ChooseC1A2L3:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ ChooseC1 | SubAbility$ SacC1A2L3 | SpellDescription$ Creature,Artifact,Land
-SVar:ChooseL1A2C3:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ ChooseL1 | SubAbility$ SacL1A2C3 | SpellDescription$ Land,Artifact,Creature
+SVar:ChooseC1L2A3:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ ChooseC1 | SubAbility$ SacC1L2A3 | SpellDescription$ Creature, Land, Artifact
+SVar:ChooseL1C2A3:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ ChooseL1 | SubAbility$ SacL1C2A3 | SpellDescription$ Land, Creature, Artifact
+SVar:ChooseA1L2C3:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ ChooseA1 | SubAbility$ SacA1L2C3 | SpellDescription$ Artifact, Land, Creature
+SVar:ChooseA1C2L3:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ ChooseA1 | SubAbility$ SacA1C2L3 | SpellDescription$ Artifact, Creature, Land
+SVar:ChooseC1A2L3:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ ChooseC1 | SubAbility$ SacC1A2L3 | SpellDescription$ Creature, Artifact, Land
+SVar:ChooseL1A2C3:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ ChooseL1 | SubAbility$ SacL1A2C3 | SpellDescription$ Land, Artifact, Creature
```

Comment:
@tool4ever @Northmoc we might need to think about a cleaner way than using that many lines

Reply:
Hmn, I don't really think it matters too much, seems this is a one-time thing?

Reply:
It does look like a fair challenge: according to the history it hasn't changed much since it was submitted in 2013.

At least, FWIW, the game has no issue displaying the corrected `SpellDescription$` items. 
![image](https://github.com/Card-Forge/forge/assets/45150760/29d544d0-b6df-4407-aace-7b355506557d)

Reply:
i mean yeah, the Effect might work

my concern was more about it if we could simplify the code somehow